tests/e2e/conftest.py

```diff
@@ -60,25 +74,15 @@ def executor() -> Executor:
 
 
 @pytest.fixture(scope="module")
-def project(request):
-    # Assume the project name and API token are set in the environment using the standard
-    # NEPTUNE_PROJECT and NEPTUNE_API_TOKEN variables.
-    #
-    # We also allow overriding the project name per module by setting the
-    # module-level `NEPTUNE_PROJECT` variable.
-    project_identifier = getattr(request.module, "NEPTUNE_PROJECT", None)
+def project():
+    project_identifier = os.getenv("NEPTUNE_E2E_PROJECT")
     if project_identifier is None:
-        project_identifier = os.getenv("NEPTUNE_PROJECT")
-        if project_identifier is None:
-            raise ValueError(
-                "Project identifier not provided. Set NEPTUNE_PROJECT environment variable or "
-                "define it in the module using `NEPTUNE_PROJECT = 'your_project_name'`."
-            )
+        raise RuntimeError("NEPTUNE_E2E_PROJECT environment variable is not set")
     return Project(project_identifier=project_identifier)
```

Comment:
Might be not worth it, but I believe of the Project class, we only ever use the "project_identifier" across the tests, so **maybe** just return string (and return the fixture)